tests/test_tankmodel.py

```diff
@@ -0,0 +1,45 @@
+
+# import pytest
+from pyfrc.physics import tankmodel, motor_cfgs
+from pyfrc.physics.units import units
+from math import degrees, radians, isclose
```

Comment:
I would personally prefer just calling it `math.isclose()`; it's clearer where it's from, and it's short anyway.